src/dynamic_graph/sot/pyrene/prologue.py

```diff
@@ -17,6 +17,11 @@
 
 from dynamic_graph.entity import PyEntityFactoryClass
 from dynamic_graph.sot.pyrene.robot import Robot
+# sys.argv is not defined when running the remove interpreter, but it is
+# required by rospy
+import sys
+if not hasattr (sys, 'argv'):
+    sys.argv = "dynamic_graph"
```

Comment:
Normally, `sys.argv` is a list, where element 0 is the name of the python main file itself. So, I think `sys.argv = ["dynamic_graph"]` would be better

Reply:
You are right, I will push the modification.